content/en/graphing/functions/beta.md

```diff
@@ -5,13 +5,69 @@ kind: documentation
+| Function         | Description                             | Example                          |
+| ---------------- | --------------------------------------- | -------------------------------- |
+| `default_zero()` | Adds a default value to sparse metrics. | `default_zero(system.load.1{*})` |
 
-**default_value**: The value to use when data is not present
+The `default_zero` function fills empty intervals using interpolation (if interpolation is enabled, which is the default for `GAUGE` type metrics) or the value 0. Like most functions, it is evaluated **after** [time and space aggregation][1].
+
+### Use Cases
+
+The `default_zero` function is intended to address the following use cases (though it may also work for other use cases):
+
+- Aligning gauges as 0 when performing arithmetic on sparse metrics (note: `COUNT` or `RATE` type metrics queried `as_count()` or `as_rate()` are _always_ aligned as 0, so using `default_zero` does not change how they are aligned; it only affects `GAUGE` type metrics).
+- Resolving monitors from a no-data condition. This works for both simple and multi-alerts, but the value 0 must not cause the monitor to trigger. For example, this would not work for a monitor with the query `avg(last_10m):avg:system.cpu.idle{*} < 10` because this monitor triggers (instead of resolving) when it evaluates to 0. Avoid using this function for [error rate monitors with `as_count()` queries][2]; it usually isn't necessary, and it causes these monitors to use the classic eval path.
+- Filling in empty intervals in sparse (but nonempty) series for visual reasons or to affect the min/max/average of a timeseries in a monitor evaluation.
+- Showing the value 0 on the query value widget when there is no data.
+
+### Example
+
+To demonstrate how the `default_zero` function works, consider this single point created for a custom metric [using DogStatsD][3]:
+
+```
+$ echo -n "custom_metric:1|g" | nc -4u -w0 127.0.0.1 8125
+```
+
+When this metric is queried over the past 30 minutes, there is a single timestamp, because only one of the query's rollup intervals has a point:
+
+```
+avg:custom_metric{*}
+
++---------------------+---------------+
+| Timestamp           | custom_metric |
++---------------------+---------------+
+| ---------           | ---------     |
+| 2019-04-16 17:56:00 | 1             |
++---------------------+---------------+
+```
+
+The `default_zero` function interpolates this point five minutes forward in time (the default limit for gauges), then fills the remaining empty intervals with zeros:
+
+```
+default_zero(avg:custom_metric{*})
+
++---------------------+-----------------------------+
+| Timestamp           | default_zero(custom_metric) |
++---------------------+-----------------------------+
+| ---------           | ---------                   |
+| 2019-04-16 17:45:00 | 0                           |
+| 2019-04-16 17:46:00 | 0                           |
+...
+| 2019-04-16 17:56:00 | 0                           |
```

Comment:
I think this value should also be 1

Reply:
Good catch. Not sure what happened there. I recreated the example and it looks good now.